### DISCUSSION TITLE
Custom key for **belongsTo** relationship is not working.

### DISCUSSION THREAD

**Discussion by w99910** [jenssegers/laravel-mongodb#2281] [Q&A]
- Laravel-mongodb Version: ^3.8
- PHP Version: 8.0
- Database Driver & Version: mongodb driver - 3.6 

### Description:
I'm trying to implement **belongsTo** relationship with custom local key and foreign key.

### Collection structure

**posts**
-  _id
-  accountId

**alerts**
- _id
- postId
- type

### Expected behaviour
Here we will match **accountId** in **posts** collection and **postId** in **alerts** collection.

**App\Models\Post**
```php
public function alert(){
         return $this->belongsTo(App\Models\Alert::class,'postId','accountId');
     }
```

### Actual behaviour
When I dd `App\Models\Alert::first()->alert` , it returns `null`.

**Comment by syafiquel** [comment]
@w99910 are you able to solve it?
